FirebaseRemoteConfig/Tests/Unit/RCNRemoteConfigTest.m

```diff
@@ -1415,6 +1419,29 @@ - (void)testSetFetchTimeoutConfigSetting {
   }
 }
 
+- (void)testFetchRequestWithUserProperties {
+  NSDictionary *userProperties = @{@"user_key" : @"user_value"};
+  NSString *req = [_settings nextRequestWithUserProperties:userProperties];
+
+  XCTAssertTrue([req containsString:@"analytics_user_properties:{\"user_key\":\"user_value\"}"]);
+}
+
+- (void)testFetchRequestWithFirstOpenTimeAndUserProperties {
+  NSDictionary *userProperties = @{@"_fot" : @1649116800000, @"user_key" : @"user_value"};
+  NSString *req = [_settings nextRequestWithUserProperties:userProperties];
+
+  XCTAssertTrue([req containsString:@"first_open_time:'2022-04-05T00:00:00Z'"]);
+  XCTAssertTrue([req containsString:@"analytics_user_properties:{\"user_key\":\"user_value\"}"]);
+}
+
+- (void)testFetchRequestFirstOpenTimeOnly {
+  NSDictionary *userProperties = @{@"_fot" : @1650312375543};
```

Comment:
The returned first_open_time won't be this precise. An expected value would be 1650315600000, which would have ISO 8601 string representation "2022-04-18T21:00:00Z".

Reply:
updated fot value in test